packages/primitives-core/types/common.d.ts

```diff
@@ -0,0 +1,41 @@
+// Definitions by:
+// Junyoung Clare Jang <https://github.com/Ailrun>
+// Sam Pettersson <https://github.com/iamsamwhoami>
+// TypeScript Version: 2.3
+
+import { ViewStyle, TextStyle, ImageStyle } from 'react-native';
+
+import {
+  Emotion,
+  ArrayInterpolation as BaseArrayInterpolation,
+  ClassInterpolation
+} from 'create-emotion';
+
+export interface StyledOptions {
+  shouldForwardProp?: (name: string) => boolean;
+}
+
+export type BasePrimitivesStyle = ViewStyle | TextStyle | ImageStyle;
+
+export type BaseInterpolation =
+  | undefined
+  | null
+  | boolean
+  | string
+  | number
+  | TemplateStringsArray
+  | BasePrimitivesStyle
+  | BaseArrayInterpolation
+  | ClassInterpolation;
```

Comment:
This one also make similar problem, since it refers `Interpolation` of `create-emotion` as the type for `__emotion_styles`.